File: C_module/Assignment 9/Q9.c
#include <stdio.h>

void reverse(int*,int);
void main() {
    int n;
    printf("Size of array :: ");
    scanf("%d",&n);
    int arr[n];
    printf("Enter array Elements :: ");
    for(int i=0;i<n;i++){
        scanf("%d",&arr[i]);
    }
    printf("Array Elements :: ");
    for(int i=0;i<n;i++){
        printf("%d ",arr[i]);
    }
    reverse(arr,n);
}

void reverse(int* arr,int n){
    int left=0,right=n-1;
    while (left<right)
    {
        int temp = arr[left];
        arr[left] = arr[right];
        arr[right] = temp;
        left++;
        right--;
    }
    printf("\nArray Elements Reversed :: ");
    for(int i=0;i<n;i++){
        printf("%d ",arr[i]);
    }
}

File: Core_Java_module/Practice/9-16-2025 (Array Practice)/Q9.java
// Reverse the given array.

import java.util.Scanner;

class Q9 {
    public static void main(String[] args) {
        int n;
		System.out.print("Enter array size : ");
		Scanner sc=new Scanner(System.in);
		n=sc.nextInt();
		int [] arr=new int [n];
		for(int i=0;i<n;i++){
			System.out.print("Enter "+(i+1)+" Element of Array : ");
			arr[i]=sc.nextInt();
		}		
		System.out.print("Array Elements are :");
		for(int i=0;i<n;i++){
			System.out.print(arr[i]+" , ");
		}
		System.out.println();
        int front=0,back=n-1;
        while(front<back){
            int temp=arr[front];
            arr[front]=arr[back];
            arr[back]=temp;
            front++;
            back--;
        }
        System.out.print("Array Elements are :");
		for(int i=0;i<n;i++){
			System.out.print(arr[i]+" , ");
		}
    }
}